src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinNamesAnnotationIntrospector.kt

```diff
@@ -120,43 +115,7 @@ internal class KotlinNamesAnnotationIntrospector(
         }
     }
 
-    @Suppress("UNCHECKED_CAST")
-    private fun findKotlinParameterName(param: AnnotatedParameter): String? {
-        return if (param.declaringClass.isKotlinClass()) {
-            val member = param.owner.member
-            if (member is Constructor<*>) {
-                val ctor = (member as Constructor<Any>)
-                val ctorParmCount = ctor.parameterTypes.size
-                val ktorParmCount = try { ctor.kotlinFunction?.parameters?.size ?: 0 }
-                catch (ex: KotlinReflectionInternalError) { 0 }
-                catch (ex: UnsupportedOperationException) { 0 }
-                if (ktorParmCount > 0 && ktorParmCount == ctorParmCount) {
-                    ctor.kotlinFunction?.parameters?.get(param.index)?.name
-                } else {
-                    null
-                }
-            } else if (member is Method) {
-                try {
-                    val temp = member.kotlinFunction
-
-                    val firstParamKind = temp?.parameters?.firstOrNull()?.kind
-                    val idx = if (firstParamKind != KParameter.Kind.VALUE) param.index + 1 else param.index
-                    val parmCount = temp?.parameters?.size ?: 0
-                    if (parmCount > idx) {
-                        temp?.parameters?.get(idx)?.name
-                    } else {
-                        null
-                    }
-                } catch (ex: KotlinReflectionInternalError) {
-                    null
-                }
-            } else {
-                null
-            }
-        } else {
-            null
-        }
-    }
```

Comment:
Looking at the previous implementation of `AnnotatedParameter.hasRequiredMarker` in `KotlinAnnotationIntrospector`, parameter acquisition was called without a check.
In other words, these check processes are now considered meaningless and have been removed.
https://github.com/FasterXML/jackson-module-kotlin/pull/760/files#diff-938d5c566367fbca40b3ef96c99967dba4cd563f3854370272e0ae6b5d0f7f0d